examples/demo/app/main.go

```diff
@@ -48,12 +49,12 @@ func initProvider() func() {
 		otelAgentAddr = "0.0.0.0:55680"
 	}
 
-	exp, err := otlp.NewExporter(
-		otlp.WithInsecure(),
-		otlp.WithAddress(otelAgentAddr),
-		otlp.WithGRPCDialOption(grpc.WithBlock()), // useful for testing
-	)
-	handleErr(err, "failed to create exporter")
+	exp, err := otlp.NewExporter(ctx, otlpgrpc.NewDriver(
+		otlpgrpc.WithInsecure(),
+		otlpgrpc.WithEndpoint(otelAgentAddr),
+		otlpgrpc.WithDialOption(grpc.WithBlock()), // useful for testing
+	))
+	handleErr(err, "failed to create resource")
```

Comment:
Is this change intentional? It appears that the previous line is exporter creation, not resource creation.

Reply:
Good catch. Fixed